stores/identity.ts

```diff
@@ -70,6 +93,30 @@ export const useIdentityStore = defineStore('identity', {
         ? state.auth.balance[urlPrefix.value] || '0'
         : '0'
     },
+    getTotalUsd: (state) => {
+      if (
+        state.multiBalances &&
+        Object.values(state.multiBalances?.chains || 0).length > 0
+      ) {
+        return Object.values(state.multiBalances.chains)
+          .flatMap((chain) => Object.values(chain))
+          .reduce((total, token) => total + parseFloat(token.usd), 0)
+      }
+
+      return 0
+    },
+    getStatusMultiBalances: (state) => {
+      let sum = 0
+      for (const key in state.multiBalances.chains) {
+        if (
+          Object.prototype.hasOwnProperty.call(state.multiBalances.chains, key)
```

Comment:
```suggestion
state.multiBalances.chains.hasOwnProperty(key)
```

Reply:
previously I'm using `state.multiBalances.chains.hasOwnProperty(key)` but the deepsource didnt happy about that :(